### DISCUSSION TITLE
Turbo cache broken in 1.4.7 - 1.5.1

### DISCUSSION THREAD

**Discussion by benmvp** [vercel/turborepo#2095] [Ideas]
I wasn't quite sure where to put this, but I banged my head for several hours trying to figure out why my Turbo cache was broken after it had been working in the past. And then I accidentally updated to `1.5.3` (not realizing it even existed because there are no release notes) and everything was fixed.

I couldn't tell you what in those releases caused it to be broken, or what was fixed in `1.5.3`. 🤷🏾 

I'm using npm workspaces. Hopefully, this saves someone some debugging time.

**Comment by nathanhammond** [comment]
We had a cache input sorting issue during those versions. This is resolved now. Thank you for the report!

**Reply by benmvp** [reply]
Thanks for fixing! 🙏🏾
